homeassistant/components/rest/__init__.py

```diff
@@ -39,7 +39,15 @@
 from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
 
-from .const import COORDINATOR, DOMAIN, PLATFORM_IDX, REST, REST_DATA, REST_IDX
+from .const import (  # noqa: F401
+    COORDINATOR,
+    DEFAULT_VERIFY_SSL,
```

Comment:
We want to make `DEFAULT_VERIFY_SSL` available to other platforms that rely on `rest` integration, so importing it here with `noqa: F401`

Reply:
You are right, I'll keep this for a follow-up PR